libkbfs/journal_server.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"sync"
+
+	"github.com/keybase/client/go/logger"
+
+	"golang.org/x/net/context"
+)
+
+type tlfJournalBundle struct {
+	// TODO: Fill in with a block journal and an MD journal.
+}
+
+// JournalServer is the server that handles write journals. It
+// interposes itself between BlockServer and MDOps. It uses MDOps
```

Comment:
"between" -> "in front of"?  The current way makes it sounds like it is a layer between `BlockServer` and `MDOps`.